pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -0,0 +1,183 @@
+package consensus
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils"
+)
+
+var info = capabilities.MustNewCapabilityInfo(
+	"ocr3",
+	capabilities.CapabilityTypeConsensus,
+	"OCR3 consensus exposed as a capability.",
+	"v1.0.0",
+)
+
+type capability struct {
+	services.StateMachine
+	capabilities.CapabilityInfo
+	store  *store
+	stopCh chan struct{}
+	wg     sync.WaitGroup
+
+	clock clockwork.Clock
+
+	newExpiryWorkerCh chan *request
+}
+
+func newCapability(s *store, clock clockwork.Clock) *capability {
+	o := &capability{
+		CapabilityInfo:    info,
+		store:             s,
+		newExpiryWorkerCh: make(chan *request),
+		clock:             clock,
+		stopCh:            make(chan struct{}),
+	}
+	return o
+}
+
+func (o *capability) Start(ctx context.Context) error {
+	return o.StartOnce("OCR3Capability", func() error {
+		innerCtx, _ := utils.ContextFromChan(o.stopCh)
+		o.wg.Add(1)
+		go o.loop(innerCtx)
+		return nil
+	})
+}
+
+func (o *capability) Stop(ctx context.Context) error {
+	return o.StopOnce("OCR3Capability", func() error {
+		o.stopCh <- struct{}{}
```

Comment:
A `close()` will 'broadcast' to all listeners instead of just one:
```suggestion
		close(o.stopCh)
```